dvc/command/version.py

```diff
@@ -59,12 +58,9 @@ def run(self):
                     fs_type = self.get_fs_type(repo.cache.local.cache_dir)
                     info.append(f"Cache directory: {fs_type}")
             else:
-                logger.warning(
-                    "Unable to detect supported link types, as cache "
-                    "directory '{}' doesn't exist. It is usually auto-created "
-                    "by commands such as `dvc add/fetch/pull/run/import`, "
-                    "but you could create it manually to enable this "
-                    "check.".format(relpath(repo.cache.local.cache_dir))
+                info.append(
+                    "Cache types: "
+                    + "https://error.dvc.org/unable-to-detect-cache-type"
```

Comment:
```suggestion
                info.append(
                    "Cache types: " + error_link("unable-to-detect-cache-type")
```
error_link helper is defined in `dvc.utils`. Could you also add such a section to https://dvc.org/doc/user-guide/troubleshooting, please? There is an `edit` button at the bottom right corner.

Reply:
Done. Updated PR + created new PR for web site documentation.